File: courses/csharp-introduction/time_to_dance!.cs
// pro

using System;

public class Program
{
    public static void Main(string[] args)
    {
        double score1 = Convert.ToInt32(Console.ReadLine());
        double score2 = Convert.ToInt32(Console.ReadLine());
        double score3 = Convert.ToInt32(Console.ReadLine());

        Console.WriteLine(Average(score1,score2,score3));
    }

    // complete the method
    static double Average(double s1, double s2, double s3)
    {
        return (s1 + s2 + s3) / 3;
    }
}

File: courses/cpp-introduction/time_to_dance!.cpp
// pro

#include <iostream>
using namespace std;

// complete the method
double average(int score1, int score2, int score3) {
    double avg = (score1 + score2 + score3) / 3;
    return avg;
}

int main() {
    double score1, score2, score3;
    cin >> score1 >> score2 >> score3;

    double result = average(score1, score2, score3);

    cout << result;

    return 0;
}